src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -720,6 +720,16 @@ def test_non_extension_redirects_past_to_details(self):
         response = self.client.get(self.url)
         self.assert3xx(response, self.next_url)
 
+    def test_source_submission_notes_not_shown_by_default(self):
+        response = self.post(has_source=False, source=None)
+        doc = pq(response.content)
+        assert 'Remember: ' not in doc('.source-submission-note').text()
+
+    def test_source_submission_notes_shown(self):
+        response = self.post(has_source=True, source=self.get_source())
+        doc = pq(response.content)
+        assert 'Remember: ' in doc('.source-submission-note').text()
```

Comment:
mmm, really these tests should be in the `DetailsPage` classes because they're testing the content on that step.